README.md

```diff
@@ -1,27 +1,130 @@
-
 # Stencil Select
 
-A styleable, accessible `<select>` component.
+A styleable, accessible `<select>` component for [Mobify’s AdaptiveJS framework](http://adaptivejs.mobify.com/).
 
-[Link to demo](#)
+**[View demo](#)**
 
 ## Requirements
 
-- Sass
-- AMD loader
-- Dust templating
+- AdaptiveJS 2.1 or greater
 
 ## Installation
 
-Installation instructions. Be as brief as possible without leaving out necessary requirements.
+```shell
+cd my-adaptive-project
```

Comment:
Pretty nitpicky, but I feel like slash delimited paths for shell commands like this are a bit clearer. `cd my/adaptive/project`. Not a big deal though.